integration-test/pom.xml

```diff
@@ -167,31 +167,25 @@
     <dependency>
       <groupId>org.apache.spark</groupId>
       <artifactId>spark-core_${scala.binary.version}</artifactId>
+      <scope>test</scope>
```

Comment:
I think these dependencies are used by code under src, their scope shoudln't be test.